app/controllers/contributors_controller.rb

```diff
@@ -5,7 +5,7 @@ class ContributorsController < ApplicationController
 
   def index
     @contributors = User.with_role(Role::CONTRIBUTOR_ROLES).includes(roles: :team).
-      where("teams.season_id IS NULL OR teams.season_id = ?", Season.current).
+      where("teams.season_id IS NULL OR teams.season_id = ? and teams.kind IS NOT NULL", Season.current).
```

Comment:
Hm, you may have found a source code artifact, @F3PiX – I don't recall having assigned anyone to a team with these special names. The _role_ is what makes an organizer:

```ruby
carpmeister.teams.pluck(:name)
# => ["joda", "RubyCats"]

carpmeister.roles.pluck(:name)
# => ["developer", "organizer", "mentor", "coach", "coach", "mentor"]